new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -5,17 +5,19 @@
 
 import java.io.File;
 
+import static oracle.weblogic.kubernetes.actions.ActionConstants.DOWNLOAD_DIR;
+import static oracle.weblogic.kubernetes.actions.ActionConstants.IMAGE_TOOL;
 import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
 
+
 /**
  *  Implementation of actions that download/install tools for the uses to use.
  *  NOTE: This class is a temporary solution, and may go away once we eventually
  *  install everything before the Java test starts to run.
  */
 
 public class Installer extends InstallWITCommon {
```

Comment:
i think we should try to avoid inheritance as much as possible

Reply:
Moved the common methods into FileUtils and a new action: Command and CommandParams.

 InstallWITCommon class is removed. We can also get rid of ActionImplCommon once Vanaj's code in PR#1535 is updated.